pkg/controller/cronjob/metrics/metrics.go

```diff
@@ -0,0 +1,31 @@
+package metrics
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const CronjobControllerV2Subsystem = "cronjob_controller_v2"
```

Comment:
Nit `CronJob` to match resource name.

Reply:
Call the sub-system just `cronjob_controller` I don't want to expose to user V2 vs V1, especially that only V2 has metrics, and the other is on his way to be removed.